index.js

```diff
@@ -58,5 +58,63 @@ const main = async (options = {}) => {
 		}));
 };
 
+const RE_PID_PPID_UID_CPU_MEM = /[ \t]*(\d+)[ \t]+(\d+)[ \t]+(\d+)[ \t]+(\d+\.\d+)[ \t]+(\d+\.\d+)[ \t]+/;
```

Comment:
Can you add a TODO about using named capture groups when targeting Node.js 10? Would make this much more readable.

For now I think a comment explaining what it is supposed to match will do.

```suggestion
const psOutputRegex = /[ \t]*(\d+)[ \t]+(\d+)[ \t]+(\d+)[ \t]+(\d+\.\d+)[ \t]+(\d+\.\d+)[ \t]+/;
```